src/Access/MemoryAccessStorage.h

```diff
@@ -30,6 +30,7 @@ class MemoryAccessStorage : public IAccessStorage
     /// Sets all entities at once.
     void setAll(const std::vector<AccessEntityPtr> & all_entities);
     void setAll(const std::vector<std::pair<UUID, AccessEntityPtr>> & all_entities);
+    void setAll(const std::vector<std::pair<UUID, AccessEntityPtr>> & all_entities, bool notify);
```

Comment:
Had to add this to use in `DiskAccessStorage::readLists()`. Subscribers should not receive notifications when entity is not loaded from disk yet